docs/en/advanced-concepts/wda-custom-server.md

```diff
@@ -61,11 +57,17 @@ so you'll see the icon of WebDriverAgentRunner application on the springboard.
 
 WebDriverAgent application acts as a REST server, which proxies external API requests to native XCTest calls
 for your application under test. The server address will be _localhost_ if you run your tests on Simulator
-or the actual phone IP address in case of real device. We use _iproxy_ to route network
-requests to a real device from _localhost_ via USB, which means one can use this tool to unify
-WDA network address for Simulator and for real device.
+or the actual phone IP address in case of real device.
+We use [appium-ios-device](https://github.com/appium/appium-ios-device) to route network requests
+to a real device from _localhost_ via USB, which means one can use this tool to unify WDA network
+address for Simulator and for real device.
+
+Alternatively, we can use _iproxy_ to handle WebDriverAgent outside Appium
```

Comment:
@KazuCocoa What do you mean there? We don't use any iproxy in the xcuitest driver right now

Reply:
This page has an example to build WDA and install it outside Appium create session. So, I leave iproxxy for the following example Java cpde.
Appium connects to the WDA as `webDriverAgentUrl` capability.

Reply:
Gotcha. We can also just connect to a remote device over `appium-ios-device`